plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/inner-blocks/taxonomy-filter/edit.tsx

```diff
@@ -94,6 +194,21 @@ const Edit = ( props: EditProps ) => {
 			</div>
 		);
 
+	if ( ! isLoading && ! isPreview && taxonomyTerms.length === 0 )
+		return (
+			<div { ...innerBlocksProps }>
+				<TaxonomyFilterInspectorControls { ...props } />
+				<Notice>
+					<p>
+						{ __(
+							'There are no products with the selected taxonomy.',
+							'woocommerce'
+						) }
```

Comment:
I wanted to see this notice and it's hard to recognise what is it about among other filters:

<img width="522" height="277" alt="image" src="https://github.com/user-attachments/assets/5cf615ae-c4f6-46ad-9396-cfa5c0507dd6" />

How about mentioning what category is that (or other value if `taxonomy` is not human readable)

```suggestion
							sprintf(
					          // translators: %s: Taxonomy name.
					          __( 'There are no products in %s taxonomy.', 'woocommerce' ),
					          taxonomy
			                 )
```

Reply:
Nice catch! I fixed it in [d9b0409](https://github.com/woocommerce/woocommerce/pull/59609/commits/d9b0409d7363a6129a9e1ed503e0a7411d7bbb99)